app/javascript/githubRepos/__tests__/singleRepo.test.jsx

```diff
@@ -58,10 +58,10 @@ describe('<SingleRepo />', () => {
     );
 
     getByText('dev.to');
-    getByText('SELECT');
+    getByText('Select');
     getByText('fork');
 
-    const removeButton = queryByText('REMOVE');
+    const removeButton = queryByText('Remove');
```

Comment:
No need to change anything, but just a heads up in case you weren't aware. You can use `queryByText` and ignore the case of the text you're checking for, e.g. `queryByText(/remove/i)`. More info in the official [getByText](https://testing-library.com/docs/dom-testing-library/api-queries#bytext) documentation.

Reply:
Gotcha, thanks!